frontend/src/components/layout/Tabs.vue

```diff
@@ -13,9 +13,14 @@
       </div>
     </div>
 
-    <div class="tab-content w-full p-16" data-test="tab-content">
+    <div class="tab-content w-full" data-test="tab-content">
       <KeepAlive>
-        <component :is="activeTab.content" v-if="activeTab" :key="activeTab.label" />
+        <component
+          :is="activeTab.content"
+          v-if="activeTab"
+          :key="activeTab.label"
+          class="bg-teal p-16 h-full"
```

Comment:
We should consider to add a comment here. It is kinda special and important why we put the classes here. As there are no tests for such things, it is easy to mess it up without recognizing and then having a hard time to find the issue what change broke it.

Reply:
:heavy_check_mark: